lib/spack/docs/packaging_guide.rst

```diff
@@ -880,6 +882,11 @@ Git fetching supports the following parameters to ``version``:
 * ``tag``: Name of a tag to fetch.
 * ``commit``: SHA hash (or prefix) of a commit to fetch.
 * ``submodules``: Also fetch submodules recursively when checking out this repository.
+* ``full_depth``: If the version of git and the specified transport
```

Comment:
Suggest describing what it does when `True` first, then explain the alternative.

Reply:
Fair enough.